### PR TITLE
Fix: handle 404 errors on fetching github issue comments page

### PR DESCRIPTION
**Pull request by philipperolet**
Related [error thread](https://dust4ai.slack.com/archives/C05F84CFP0E/p1699882670861289)

### CODE REVIEW

**Comment by philipperolet** [comment]
No need for a queue bump for this

File: connectors/src/connectors/github/temporal/activities.ts [hunk 1]
```diff
@@ -106,13 +106,23 @@ export async function githubUpsertIssueActivity(
...
+      comments = await getIssueCommentsPage(
+        installationId,
+        repoName,
+        login,
+        issueNumber,
+        resultPage
+      );
+    } catch (e: any) {
+      if (e.status === 404) {
+        // Github may return a 404 on the first page if the issue has no comments
```
Comment (fontanierh) [inline connectors/src/connectors/github/temporal/activities.ts:120]: I hope we won't run into cases where we get transient 404s

**Review by fontanierh — DISMISSED** [review]
🙏

File: connectors/src/connectors/github/temporal/activities.ts [hunk 2]
```diff
@@ -106,13 +106,23 @@ export async function githubUpsertIssueActivity(
...
+    let comments = undefined;
+    try {
+      comments = await getIssueCommentsPage(
+        installationId,
+        repoName,
+        login,
+        issueNumber,
+        resultPage
+      );
+    } catch (e: any) {
```
Comment (spolu) [inline connectors/src/connectors/github/temporal/activities.ts:118]: Warning should be removed before merging :pray:

Comment (philipperolet) [reply connectors/src/connectors/github/temporal/activities.ts:118]: ack

File: connectors/src/connectors/github/temporal/activities.ts (hunk 1 above)
Comment (philipperolet) [reply connectors/src/connectors/github/temporal/activities.ts:120]: So do I!

**Pull request merged**